src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -906,6 +906,31 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
             }
         }
 
+        [Fact]
```

Comment:
This be marked Outerloop similar to the rest of the tests, especially since it uses the Azure test server endpoint.